test/parallel/test-process-execve-fail.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const { isMainThread, mustCall, skip, isWindows } = require('../common');
+const { throws } = require('assert');
+
+if (!isMainThread) {
+  skip('process.execve is not available in Workers');
+}
+
+if (!isWindows) {
+  // Non existent path name
+  {
+    throws(mustCall(() => {
```

Comment:
Passing a must call into `throws` doesn't make a lot of sense in my opinion. We know that `throws` is goingt to call that function immediately (we have tests proving this).

Reply:
Thanks, I removed that!